### PR TITLE
SqlServer LocalDb on Windows 7

### PR DESCRIPTION
**Pull request by dvdstelt**
Users reported issues with LocalDb on Windows 7. I think the main reason is because SQL2016 doesn't support Windows 7.
You should therefor download & install SQL Express 2014 which contains LocalDb as well.

I should verify this before merging this, but maybe you know this is true and then you can just merge this ;-)

### CODE REVIEW

**Comment by mauroservienti** [comment]
We don’t really have any constraint on the LocalDB version, thus whichever works on the user machine should work for us, I guess.

**Comment by adamralph** [comment]
Marking as WIP due to:

> I should verify this before merging this, but maybe you know this is true and then you can just merge this ;-)

File: README.md [hunk 1]
```diff
@@ -179,6 +179,10 @@ Using an elevated PowerShell prompt, run `Teardown-Databases.ps1` followed by `S
 
 Connect to the instance and run `Teardown-Databases.sql` followed by `Setup-Databases.sql`, both located in the [exercises/scripts](exercises/scripts) folder.
 
+### How can I use SQL Server LocalDb on Windows 7
+
+Windows 7 isn't a supported operating system for SQL Server 2016. Instead install SQL Server 2014 Express, which you can (download here)[https://www.microsoft.com/en-us/download/details.aspx?id=42299].
```
Comment (adamralph) [inline README.md]: > Windows 7 isn't a supported operating system for SQL Server 2016

The connection of this sentence to:

> How can I use SQL Server LocalDb on Windows 7

is not obvious.

Also, the text does not answer the question. It says you should use SQL Server 2014 Express instead. The title should probably be more general, e.g.

> Using Windows 7

**Comment by adamralph** [comment]
@Particular/workshop-maintainers @dvdstelt I added a commit which generalises the FAQ. Please review.

**Pull request merged**